core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.html

```diff
@@ -48,40 +48,36 @@
                 <div class="field">
                     <label
                         *ngIf="requiredPrompt; else labelNotRequired"
-                        dotFieldRequired
                         data-testid="prompt-label"
+                        dotFieldRequired
                         for="prompt">
                         {{ promptLabel | dm }}
                     </label>
                     <textarea
                         class="w-full ai-image-prompt__textarea"
                         id="prompt"
                         [placeholder]="promptTextAreaPlaceholder | dm"
-                        rows="11"
                         cols="30"
+                        formControlName="text"
                         pInputTextarea
-                        formControlName="text">
+                        rows="11">
                     </textarea>
                 </div>
                 <div class="field" *ngIf="aiProcessedPrompt && !isLoading">
                     <label class="ai-image-prompt__rewritten-label" for="prompt-rewritten"
                         >{{ 'block-editor.extension.ai-image.rewritten' | dm }}
-                        <i
-                            class="pi pi-copy"
-                            [pTooltip]="tooltipText"
-                            (click)="copyToClipboard()"
-                            tooltipPosition="bottom"
-                            hideDelay="800"
-                            data-testid="copy-to-clipboard"></i>
+                        <dot-copy-button
```

Comment:
we need to move that to the text area as requested by desing. 
<img width="309" alt="image" src="https://github.com/dotCMS/core/assets/31667212/05d41c94-7cb8-4b7d-8ce2-3bcf0d53eb4c">